src/Loader.php

```diff
@@ -333,7 +331,7 @@ public function get_twig()
          *  *
          *  * @link https://twig.symfony.com/doc/2.x/api.html#environment-options
          *  *
-         *  * @param array $options An array of environment options.
+         *  * \@param array $options An array of environment options.
```

Comment:
Why the docblock backslash?

Reply:
@nlemoine I think I added this because this is a DocBlock inside an `@example` tag. If I remember correctly, this lead to problems when generating the documentation when I didn’t use the backslashes.

Reply:
I was just curious about this :)